tools/topology/sof/pipe-eq-iir-volume-capture.m4

```diff
@@ -79,6 +81,17 @@ C_CONTROLBYTES(DEF_EQIIR_COEF, PIPELINE_ID,
 	,
 	DEF_EQIIR_PRIV)
 
+# EQ enum list
+CONTROLENUM_LIST(eq_preset, LIST(`	', `"preset1"', `"preset2"'))
```

Comment:
This would be better as DEF_EQ_PRESET with pipeline number appended to eq_preset_. The enum items count seems to grow with number of instances.